src/tool/subcommands/archive_cmd.rs

```diff
@@ -171,12 +177,32 @@ impl ArchiveCommands {
                 let variant = store.variant().to_string();
                 let heaviest = store.heaviest_tipset()?;
                 let index_size_bytes = store.index_size_bytes();
+                let snapshot_version = if let Some(metadata) = store.metadata() {
+                    metadata.version
+                } else {
+                    FilecoinSnapshotVersion::V1
+                };
                 println!(
                     "{}",
-                    ArchiveInfo::from_store(&store, variant, heaviest, index_size_bytes)?
+                    ArchiveInfo::from_store(
+                        &store,
+                        variant,
+                        heaviest,
+                        snapshot_version,
+                        index_size_bytes
+                    )?
                 );
                 Ok(())
             }
+            Self::Metadata { snapshot } => {
+                let store = AnyCar::try_from(snapshot.as_path())?;
+                if let Some(metadata) = store.metadata() {
+                    println!("{metadata}");
+                } else {
+                    println!("No FRC-0108 metadata found in the snapshot");
```

Comment:
I think we can have a better response here.

 couple of suggestion:
- "This snapshot uses the v1 format (no metadata header)")
- "No metadata found (required by v2 snapshot) - this appears to be a v1 snapshot");